cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -270,9 +268,11 @@ private void stopQuietly(Service service) {
   /**
    * Stops a guava {@link Service}. No exception will be thrown even stopping failed.
    */
-  private void stopQuietly(TwillRunnerService service) {
+  private void stopQuietly(@Nullable TwillRunnerService service) {
```

Comment:
It can be null if during the leader() transition there is exception, causing the `TwillRunner` not instantiated.